.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
If it's not yet implemented, I wonder why it's working for us in Insights. @benpatterson added that config - Ben, thoughts?